tidb-lightning/tidb-lightning-physical-import-mode.md

```diff
@@ -92,3 +92,9 @@ It is recommended that you allocate CPU more than 32 cores and memory greater th
 - When you use TiDB Lightning with TiCDC, note the following:
 
     - TiCDC cannot capture the data inserted in the physical import mode.
+
+- When you use TiDB Lightning with BR, note the following:
+
+    - Snapshot backups by BR of tables being imported by Lightning may result in inconsistent backup data for those tables.
+    - BR performing backups based on AWS EBS volume snapshots may lead to data import failures via Lightning.
+    - Point-in-Time Recovery (PiTR) is incapable of backing up data imported by Lightning.
```

Comment:
```suggestion
    - When BR backs up snapshots of tables that are being imported by TiDB Lightning, it might result in inconsistent backup data for those tables.
    - When BR backs up based on AWS EBS volume snapshots, it might lead to data import failure by TiDB Lightning.
    - Point-in-Time Recovery (PiTR) can not back up data imported by TiDB Lightning.
```